api/src/database-units/alert_severity.ts

```diff
@@ -0,0 +1,10 @@
+import { z } from 'zod';
+
+/**
+ * Alert Severity Data Type.
+ *
+ * @description Data type for `alert_severity`.
+ */
+export const AlertSeverity = z.enum(['info', 'warning', 'error', 'success']);
```

Comment:
question: Should we instead just export the actual enum? Slightly different pattern from the other `database-models` files, but maybe this makes sense and is more useful?

```js
enum AlertSeverity {
    info = 'info',
    warning = 'warning',
    ...
}
```

Reply:
Made some updates to this file and the similar quantitative_unit file.